ocr_space.py

```diff
@@ -4,6 +4,7 @@
 OCRSPACE_API_KEY = "" # Contact owner for developer key
 OCRSPACE_API_URL_USA = "https://apipro1.ocr.space/parse/image"
 OCRSPACE_API_URL_EU = "https://apipro2.ocr.space/parse/image"
+tesseract_cmd = "/usr/bin/tesseract"
```

Comment:
Hey Matthew, 
Full disclosure, I didn't write this code, my roommate did. I'm not a programmer, but I brought him in to help me figure this out because I recently switched to Linux and couldn't get game2text to compile (it's an important part of my studying routine, haha.)

This was intended, the original syntax was causing it to fail to compile on my system, and when he modified it and moved the tesseract line, it did compile. 

This is what he had to say about it:

"Tesseract_cmd = “usr/bin/tesseract”

Was previously:

Pytesseract.pytesseract.tesseract_cmd = “usr/bin/tesseract”

And the interpreter was not understanding the variable. So I just called it ‘tesseract_cmd”

Also. This particular variable was being called to at the end of the script. In my experience it would need to be listed at the beginning of the script. So I moved its position. And the code compiled."

However, when I tried to reproduce the error just now for you on the original build, it compiled fine and launched. (I had the error saved, but we were up until about 4am troubleshooting this the other night and accidentally closed the doc I had it pasted the error into. Sorry about that.)



The issue must have been due to a problem with the dependencies or something? I will say for me pip install -r wasn't able to parse through the entire requirements.txt, and I had to go through line by line and pip install to ensure all the dependencies were installed.

Either way it looks like it still runs regardless of whether I try to compile the original version, or the modified version now. I'll continue to try to reproduce the original error.